training/distributed_training/pytorch/model_parallel/gpt-j/11_train_gptj_smp_tensor_parallel_notebook.ipynb

```diff
@@ -39,7 +39,7 @@
     "! pip install -qU \"sagemaker>=2,<3\"\n",
```

Comment:
<p>Suggest to change the intro sentence:</p><p> </p><p>You'll learn how to train the GPT-J model with tensor parallelism on the GLUE sst2 dataset.</p><p> </p><p>--&gt;</p><p> </p><p>You'll learn how to run FP16 training of the GPT-J model with tensor parallelism on the GLUE sst2 dataset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>